panther_analysis_tool/main.py

```diff
@@ -398,6 +399,88 @@ def upload_analysis(args: argparse.Namespace) -> Tuple[int, str]:
     return 0, ""
 
 
+def confirm_analysis_exists(args: argparse.Namespace, analysis_id_list: list) -> list:
+    validation_payload = {"listDetections": {"ids": analysis_id_list, "fields": ["id"]}}
+    client = get_client(args.aws_profile, "lambda")
+
+    validation = client.invoke(
+        FunctionName="panther-analysis-api",
+        InvocationType="RequestResponse",
+        LogType="None",
+        Payload=json.dumps(validation_payload),
+    )
+
+    api_response = json.loads(validation["Payload"].read().decode("utf-8"))
+    json_response = api_response["body"]
+
+    json_output = json.loads(json_response)
+
+    if len(analysis_id_list) != json_output["paging"]["totalItems"]:
+        analysis_found = {}
+
+        for detection in json_output["detections"]:
+            analysis_found[detection["id"]] = detection
+        # Build a list of analysis we've confirmed are in Panther
+        for analysis in analysis_id_list:
+            if analysis not in analysis_found:
+                logging.info("%s was not found, skipping...", analysis)
+
+        return list(analysis_found.keys())
+
+    return analysis_id_list
+
+
+# pylint: disable=too-many-locals
+def delete_analysis(args: argparse.Namespace) -> Tuple[int, str]:
+
+    client = get_client(args.aws_profile, "lambda")
+    analysis_id_list = args.analysis_id
+    payload: dict = {"deleteDetections": {"entries": []}}
+
+    # Validate the Detection that we are deleting exists in Panther
+    # If nothing was found bail out, otherwise prepare to delete what has been confirmed
+
+    analysis_id_list = confirm_analysis_exists(args, analysis_id_list)
+    if len(analysis_id_list) == 0:
+        logging.error("No matching analysis found, exiting")
+        return 1, ""
+
+    # Get user confirmation to delete
+    analysis_id_string = " ".join(analysis_id_list)
+    logging.warning("You are about to delete detections %s", analysis_id_string)
+    confirm = input("Continue? (y/n)")
```

Comment:
Nit: add a space after the closing parentheses 
```suggestion
    confirm = input("Continue? (y/n) ")
```